config/wallet-config.json

```diff
@@ -52,6 +45,13 @@
       "hasTradingFees": true,
       "hasUnitedStatesAvailability": false
     },
+    "coinbase": {
+      "enabled": true,
+      "hasFastCheckoutProcess": false,
+      "name": "Coinbase",
+      "hasTradingFees": true,
+      "hasUnitedStatesAvailability": true
+    },
```

Comment:
Why did this move again? Are we using the object for ordering?

Reply:
We do rely on the ordering here, yep.